internal/render/worker/container.go

```diff
@@ -174,3 +175,57 @@ func renderRealMemorySlurmd(resources corev1.ResourceRequirements) int64 {
 	memoryInMebibytes := memoryInBytes / 1_048_576 // 1 MiB = 1,048,576 bytes
 	return memoryInMebibytes
 }
+
+func renderContainerSshd(
```

Comment:
I think that this part same as in login node... Can we use it one time from common?

Reply:
they have different arguments.  i do not think we have to spend  time for this way